src/reducers/tests/source-tree.spec.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+declare var describe: (name: string, func: () => void) => void;
+declare var it: (desc: string, func: () => void) => void;
+declare var expect: (value: any) => any;
+
+import update, { InitialState } from "../source-tree";
+
+describe("source tree reducer", () => {
+  it("should set the expanded state", () => {
+    const state = InitialState();
+    const expanded = new Set(["foo", "bar"]);
+    const updatedState = update(state, {
+      type: "SET_EXPANDED_STATE",
+      expanded
+    });
+
+    expect(updatedState.expanded).toBe(expanded);
+  });
+});
```

Comment:
i think this test is redundant. we get the same coverage from the action test.

Reply:
lol .. yeah ... tot so too. never written a reducer test, got excited with the tests :)

Reply:
will pull it out.